util/assertString.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Looks like it is possible with some babel tinkering

> `resolveModuleSource`	`null`	Resolve a module source ie. `import "SOURCE";` to a custom value. Called as `resolveModuleSource(source, filename)`.

https://babeljs.io/docs/usage/options/